float32 vel
bool linetrace_en
int8 crossline_cnt
---
float32 current_vel
